src/ui/components/PlaylistsNavLink/PlaylistsNavLink.tsx

```diff
@@ -1,15 +1,30 @@
 import * as React from 'react';
 import { NavLink } from 'react-router-dom';
 
+import * as app from '../../lib/app';
+import * as PlayerActions from '../../actions/PlayerActions';
 import * as styles from './PlaylistsNavLink.css';
 
+import { TrackModel, PlaylistModel } from 'src/shared/types/interfaces';
+
 interface Props {
   className?: string;
   playlistId: string;
   onContextMenu: (playlistId: string) => void;
 }
 
 export default class PlaylistsNavLink extends React.Component<Props> {
+  // Start playing playlist on double click (Issue #495)
+  handleDoubleClick = async () => {
+    try {
+      const playlist: PlaylistModel = await app.models.Playlist.findOneAsync({ _id: this.props.playlistId });
+      const tracks: TrackModel[] = await app.models.Track.findAsync({ _id: { $in: playlist.tracks } });
```

Comment:
This (all async operations like this one) should be part of an action, how about `PlaylistActions.play()` or `PlaylistActions.startPlayback()`?

https://github.com/martpie/museeks/blob/master/src/ui/actions/PlaylistsActions.ts

Reply:
Okay, great idea!